hack/build.sh

```diff
@@ -9,6 +9,22 @@ declare -a VERSIONS=(2.4)
 OS=$1
 VERSION=$2
 
+function squash {	
+	base=$(grep ^FROM Dockerfile | awk '{print $2}')
+	# install the docker layer squashing tool
+	easy_install --user pip
+	rm -rf docker-scripts
+	git clone --branch 0.3.0 https://github.com/goldmann/docker-scripts/
+	pushd docker-scripts
+	~/.local/bin/pip install --force --user .
+	popd
+	# find the top of the "centos" or "rhel" layer
+	layer=$(~/.local/bin/docker-scripts layers -m ${base} | tail -n 1)
+	# squash everything above it
+	$HOME/.local/bin/docker-scripts squash -f $layer ${IMAGE_NAME}
```

Comment:
Using `$base` here does not give you the expected results? I'm pretty sure the `squash` command will properly find the layer ID. Can you compare the output of the `squash` logs (the "Attempting to squash from layer" line with the ID you get from `layers -m $base`? These should be really the same since I do this checking: https://github.com/goldmann/docker-scripts/blob/master/docker_scripts/squash.py#L347